app/src/main/java/org/mozilla/fenix/compose/list/HomeSectionHeader.kt

```diff
@@ -0,0 +1,71 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.fenix.compose.home
+
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.wrapContentHeight
+import androidx.compose.foundation.text.ClickableText
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.semantics.contentDescription
+import androidx.compose.ui.semantics.semantics
+import androidx.compose.ui.text.AnnotatedString
+import androidx.compose.ui.text.TextStyle
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.sp
+import org.mozilla.fenix.R
+import org.mozilla.fenix.compose.SectionHeader
+import org.mozilla.fenix.theme.FirefoxTheme
+
+/**
+ * Homepage header.
+ *
+ * @param text The header string.
+ * @param description The description for click action
+ * @param onShowAllButtonClick Invoked when "Show all" button is clicked.
+ */
+@Composable
+fun HomeSectionHeader(
+    headerText: String,
+    description: String,
+    onShowAllClick: () -> Unit = {}
+) {
+    Row(
+        modifier = Modifier.fillMaxWidth(),
+    ) {
+        SectionHeader(
+            text = headerText,
+            modifier = Modifier
+                .weight(1f)
+                .wrapContentHeight(align = Alignment.Top)
+        )
+
```

Comment:
There should be a `Spacer` (maybe 16.dp?)  between the header text and show all, just in case some weird header text string or device font size forces them to butt up to each other.

Reply:
Sounds right!